util/movement-types/src/lib.rs

```diff
@@ -168,6 +168,15 @@ impl From<Commitment> for Vec<u8> {
 	}
 }
 
+impl fmt::Display for Commitment {
```

Comment:
This creates a canonical string representation for commitment values.
I think it's OK, but if it's only meant to be used in logging and debugging, it may be better to go with `Debug`.

Reply:
`Debug` is already implemented for `Commitment`, I needed to implement `Display` on it for some other reason, what that was now I can't quite recall!

Reply:
`Debug` is just a derive though, it prints decimal byte values. Maybe we should implement it to print hex too.